source/features/mentions-highlight.js

```diff
@@ -0,0 +1,13 @@
+import {getUsername} from '../libs/utils';
+
+export default () => {
+	const username = getUsername();
+
+	if (username) {
```

Comment:
Remove this `if`, per https://github.com/sindresorhus/refined-twitter/pull/43#discussion_r165831205